src/main/java/org/apache/sling/feature/cpconverter/ContentPackage2FeatureModelConverter.java

```diff
@@ -135,10 +135,6 @@ public ContentPackage2FeatureModelConverter(boolean strictValidation) {
         }
     }
 
-    public void setHandlersManager(EntryHandlersManager handlersManager) {
```

Comment:
This makes the tests not compile anymore :-)

Reply:
yes, forgot to commit the adjusted test, done now in https://github.com/apache/sling-org-apache-sling-feature-cpconverter/pull/74/commits/963bf29b3fecc262b3b74e26f0ae8b00be6b754f.

Reply:
Thanks, works now - however, there was a change in the repoinit parser that made a test fail (unrelated but as we are using the SNAPSHOT for now it impacts this PR as well). I fixed it in master so if you merge master it should start working here too.

Reply:
Done in https://github.com/apache/sling-org-apache-sling-feature-cpconverter/pull/74/commits/f61cd99afcc94010e1f20dabeb76097bd8416175.